server.go

```diff
@@ -62,16 +62,27 @@ func (s *server) Start() (bool, error) {
 		}
 	}, bufSize)
 
+	logging.Printf("Creating bucket container")
```

Comment:
Do we need this logging (as in the added logging generally, not this line in particular) long term?

Reply:
I think it's not a bad thing to have more logging, but some of these bits are a little excessive.

Maybe we can put the excessively verbose log lines at a trace level when we add leveled logging?